gpcontrib/arenadata_toolkit/arenadata_toolkit--1.3--1.4.sql

```diff
@@ -0,0 +1,357 @@
+/* gpcontrib/arenadata_toolkit/arenadata_toolkit--1.3--1.4.sql */
+
+DO $$
+BEGIN
+	IF (SELECT a.attrelid IS NULL
```

Comment:
You can do some more optimization here and in other similar places like this:
```suggestion
	IF a.attrelid IS NULL
```